rskj-core/src/main/java/co/rsk/net/discovery/PeerExplorer.java

```diff
@@ -217,6 +238,12 @@ private void handleFindNode(FindNodePeerMessage message) {
         NodeID nodeId = message.getNodeId();
         Node connectedNode = this.establishedConnections.get(nodeId);
 
+        logger.debug("handleFindNode - Handling find node message with " +
+                        "nodeId: {}, " +
+                        "messageId: {}, " +
+                        "connectedNode: {}"
+                , message.getNodeId(), message.getMessageId(), connectedNode);
+
         if (connectedNode != null) {
```

Comment:
I guess we should print a WARN here if connectedNode is null. So that we may not need the one above - `logger.debug("handleFindNode - Handling find node message with ...` - as we could extend the one bellow - `logger.debug("About to send [{}] neighbors to ip[{}] ...`

Reply:
Done